packages/fx-core/resource/package.nls.json

```diff
@@ -1022,6 +1022,8 @@
   "driver.teamsApp.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
   "driver.teamsApp.validate.invalidParameter": "Either %s or %s parameter should be provided for action %s",
   "driver.teamsApp.validate.result": "Teams Toolkit has completed checking your app package against validation rules. %s failed, %s passed. Check [output window](%s) for details.",
+  "driver.teamsApp.update.notExistsError": "Teams app with ID %s does not exist in Teams Developer Portal.",
```

Comment:
error.teams.AppIdNotExistError
error.teams.InvalidAppIdError

Reply:
please use such pattern as error key: error.<category1>.<category2 if necessary>.<Error name capitalized for first letter>